src/UnitOfWork/ReducingUnitOfWorkExecutor.php

```diff
@@ -25,9 +25,14 @@ public function __construct(UnitOfWorkExecutor $unitOfWorkExecutor, OperationCon
     }
 
 
-    public function execute(UnitOfWork $unitOfWork): void
-    {
-        $operations = $this->consolidator->consolidate($unitOfWork->getOperations());
+    public function execute(
+        UnitOfWork $unitOfWork,
+        ?OperationConsolidationMode $operationConsolidationMode = null
+    ): void {
+        $operations = $this->consolidator->consolidate(
+            $unitOfWork->getOperations(),
+            $operationConsolidationMode ?? new OperationConsolidationMode(false, false, false),
```

Comment:
8.1 should allow setting this directly
```suggestion
        OperationConsolidationMode $operationConsolidationMode = new OperationConsolidationMode(false, false, false),
    ): void {
        $operations = $this->consolidator->consolidate(
            $unitOfWork->getOperations(),
            $operationConsolidationMode,
```